### DISCUSSION TITLE
SSR on Cloudflare workers

### DISCUSSION THREAD

**Discussion by danbars** [quasarframework/quasar#13381] [Ideas / Proposals]
Cloudflare workers are pretty popular Serverless environment.
It is different then most other Serverless in that it doesn't run on node, but rather on V8 Isolate. Essentially this is a WebWorker interface. 
In order to allow SSR in this environment, there has to be no dependency on Node or Express. 
The new `production-export.js` is a step in the right direction, but I still couldn't make this work. 

It'd be great if Quasar supports SSR on Cloudflare.

**Comment by danbars** [comment]
A lot has changed since this was written. Cloudflare Pages is now mature enough, and many frameworks already have native integrations. 
I'd love to see Quasar in this list: https://developers.cloudflare.com/pages/framework-guides/

**Reply by chiacy** [reply]
Me too while exploring the possibility of quasar ssr running on cloudflare pages

**Reply by lsolano2707** [reply]
Hi, any update on deploying quasar v2 with ssr in cloudflare workers?. I have tried everything and it still doesn't work

**Reply by EthraZa** [reply]
I was about to start a new project with Quasar and CF workers by the next week or so and just came across this. I'm sad now.
No updates I guess?

**Reply by leonardo-solano-deel** [reply]
Hi, any updates?

**Reply by lsolano2707** [reply]
Hi, any updates? I see that Quasar SSR is currently starting to be supported by other providers, but Cloudflare isn't on the list yet.
